### DISCUSSION TITLE
How to add a ListingProperty?

### DISCUSSION THREAD

**Discussion by quape** [etsy/open-api#931] [Q&A] [answered]
Hi there,

I want to change the "Craft type" of a listing which could be done in v2 by adding the corresponding property_id to the listing. Unfortunately I couldn't find such a method in v3. The only thing I can do is adding a property as an inventory product to the listing. This works so that "craft type" is shown as a product variant in the listing but not as the main "craft type" of the listing itself. In my understanding the API should change the property of the listing itself if there is only one inventory, but this isn't the case.

Any ideas?
Thanks

**Comment by itiievskyi** [comment]
Hi,

If the category your listing belongs to has this property, I believe all you need is to call [updateListingProperty](https://developer.etsy.com/documentation/reference/#operation/updateListingProperty). 
You need to know property_id and its values/value ids. Property ID should be 47626759760 for Craft type. Does it make sense for you?

**Comment by quape** [comment, marked as answer]
Got it! That worked, thanks!
